.github/workflows/build_test.yml

```diff
@@ -17,40 +17,56 @@ on:
 jobs:
   ubuntu_build:
     name: Ubuntu Build ${{ matrix.name }}
-    runs-on: [ubuntu-latest]
+    runs-on: ${{ matrix.os }}
     strategy:
       matrix:
         # We have one job per "name" in the matrix. Attributes are set on the
         # specific job names.
-        name: [release, debug, asan, msan, scalar]
+        name: [release, debug, scalar, asan, msan, release-nojpeg, release-gcc8, release-clang-5]
         include:
           - name: release
+            os: ubuntu-latest
             test_in_pr: true
           - name: debug
+            os: ubuntu-latest
           # Build scalar-only hwy instructions.
           - name: scalar
+            os: ubuntu-latest
             mode: release
             cxxflags: -DHWY_DISABLED_TARGETS=~HWY_SCALAR
+          # TODO(eustas): why ASAN has no params?
+          - name: asan
+            os: ubuntu-latest
           # Disabling optional features to speed up msan build a little bit.
           - name: msan
+            os: ubuntu-latest
             cmake_args: >-
               -DJPEGXL_ENABLE_DEVTOOLS=OFF -DJPEGXL_ENABLE_PLUGINS=OFF
               -DJPEGXL_ENABLE_VIEWERS=OFF
           # Build with support for decoding to JPEG bytes disabled. Produces a
           # smaller build if only decoding to pixels is needed.
           - name: release-nojpeg
+            os: ubuntu-latest
             mode: release
             cxxflags: -DJXL_DEBUG_ON_ABORT=0
             cmake_args: >-
               -DJPEGXL_ENABLE_TRANSCODE_JPEG=OFF
               -DJPEGXL_ENABLE_PLUGINS=OFF
               -DJPEGXL_ENABLE_VIEWERS=OFF
           # Builds with gcc in release mode
-          - name: release:gcc8
+          - name: release-gcc8
```

Comment:
why the rename? in gitlab we had the compiler as `:compiler`

Reply:
Because ":" can not be part of name in flat list in yml. Rolled back.